pkg/exporter/process.go

```diff
@@ -101,7 +107,7 @@ func InitExportingProcess(input ExporterInput) (*ExportingProcess, error) {
 			if configErr != nil {
 				return nil, configErr
 			}
-			conn, err = tls.Dial(collectorAddr.Network(), collectorAddr.String(), config)
+			conn, err = tls.Dial(input.CollectorProtocol, input.CollectorAddress, config)
```

Comment:
Exporting process should always expect a DNS name for collectorAddress for TLS or is IP a possible input too?

Reply:
IP is possible, but for Antrea we support DNS name only. and the DNS name will be passed as a parameter in flow aggregator.

Reply:
Add a comment here we take the input address directly without resolving any DNS name.

Reply:
> IP is possible, but for Antrea we support DNS name only. and the DNS name will be passed as a parameter in flow aggregator.

That's not accurate I think. Someone could choose to provide an IP address, in which case the certificate would be generated using the IP address as the DNS name. This is a valid use case and it should work.

Reply:
Agree. Since we cannot get the flow aggregator IP address currently in Antrea, I only use DNS name. In the future we can support custom fields (IPs or DNSs) to generate certificate.